markhor_bringup/launch/markhor_web_ui.launch

```diff
@@ -0,0 +1,11 @@
+<launch>
+    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
+	<!--<arg name="ssl" value="true"/>-->

```

Comment:
Could you remove the commented argument if we are not going to use them ?

Reply:
Done